send-transaction-service/src/send_transaction_service.rs

```diff
@@ -821,30 +872,11 @@ mod test {
             config.batch_size,
             &stats,
         );
-        assert_eq!(transactions.len(), 1);
```

Comment:
why we don't need this code any longer?

Reply:
because we remove both transactions from hashmap in one `process_transactions` call